masl.src.html

```diff
@@ -19,10 +19,9 @@ <h2>Introduction</h2>
         Anywhere you have resources that will be deployed in real-world systems, the potential
         metadata needs of those systems are effectively unbounded. This is particularly true of
         decentralised systems that need to exhibit "web-like" behaviour: in order to have
-        reproducible and safe execution when content is sourced from arbitrary sources, it is
+        reproducible and safe execution when content is sourced from arbitrary authorities, it is
```

Comment:
```suggestion
        reproducible and safe execution when content is sourced from arbitrary origins, it is
```

The whole point is they're not authoritative…